cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/ShardedHBaseQueueStrategy.java

```diff
@@ -68,7 +69,7 @@
   // Number of bytes as the row key prefix, including salt bytes added by the row key distributor
   public static final int PREFIX_BYTES = SaltedHBaseQueueStrategy.SALT_BYTES + Bytes.SIZEOF_LONG + Bytes.SIZEOF_INT;
 
-  private static final Function<byte[], byte[]> ROW_KEY_CONVERTER = new Function<byte[], byte[]>() {
+  static final Function<byte[], byte[]> ROW_KEY_CONVERTER = new Function<byte[], byte[]>() {
```

Comment:
Can you move some of the scan logic into this class so that you don't have to turn things in this class from private to package protected? It's not a very good practice for encapsulation.